dmci/distributors/distributor.py

```diff
@@ -78,6 +79,14 @@ def __init__(self, cmd, xml_file=None, metadata_id=None, worker=None, **kwargs):
                     self._valid = False
                     return
 
+        if 'path_to_parent_list' in kwargs and kwargs['path_to_parent_list'] is not None:
```

Comment:
Is parent_list supposed to be for visible for every file going through dmci, then maybe should be set up in config-class instead of distributor.